rpm/SPECS/cntlm.spec

```diff
@@ -43,8 +45,10 @@ make %{?_smp_mflags} SYSCONFDIR=%{_sysconfdir}
 make BINDIR=%{buildroot}%{_sbindir} MANDIR=%{buildroot}%{_mandir} SYSCONFDIR=%{buildroot}%{_sysconfdir} NOSTRIP=1 install
 
 install -D -m 0644 %{SOURCE1} %{buildroot}%{_tmpfilesdir}/%{name}.conf
-install -D -m 0644 %{SOURCE2} %{buildroot}%{_unitdir}/%{name}.service
-install -D -m 0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/sysconfig/cntlmd
+install -D -m 0644 %{SOURCE2} %{buildroot}%{_sysconfdir}/sysconfig/cntlmd
+install -D -m 0644 %{SOURCE3} %{buildroot}%{_unitdir}/%{name}.service
+install -D -m 0644 %{SOURCE4} %{buildroot}%{_unitdir}/%{name}@.service
```

Comment:
On my system it seems service units that are ran for a user are in `/usr/lib/systemd/user`. I guess that may be a different variable.

Reply:
RPM macro to use per the docs of the systemd RPM macros is %{userunitdir}